### PR TITLE
Cleanup Hello OpenShift example

### PR DESCRIPTION
**Pull request by rhcarvalho**

### CODE REVIEW

File: examples/hello-openshift/README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 Hello, OpenShift!
 -----------------
 
-This example will serve an http response of "Hello OpenShift!" to [http://localhost:6061](http://localhost:6061).  To create the pod run:
+This example will serve an HTTP response of "Hello OpenShift!" to http://localhost:6061.
```
Comment (mfojtik) [inline examples/hello-openshift/README.md]: i don't think it will serve on `localhost:6061`... at least not in k8s

Comment (rhcarvalho) [inline examples/hello-openshift/README.md]: I had the same doubt, even had it changed to :8080 before opening the PR. But indeed it is available on localhost:6061.

Comment (mfojtik) [inline examples/hello-openshift/README.md]: @rhcarvalho if it runs as Pod, how I can connect to it then?

Comment (rhcarvalho) [inline examples/hello-openshift/README.md]:
```
[vagrant@openshiftdev origin]$ oc describe pod hello-openshift                                                        
Name:                           hello-openshift
Image(s):                       openshift/hello-openshift
Host:                           openshiftdev.local/127.0.0.1
Labels:                         name=hello-openshift
Status:                         Running
IP:                             172.17.0.104
Replication Controllers:        <none>
Containers:
  hello-openshift:
    Image:              openshift/hello-openshift
    State:              Running
      Started:          Mon, 22 Jun 2015 14:16:44 +0000
    Ready:              True
    Restart Count:      0
Conditions:
  Type          Status
  Ready         True 
No events.
[vagrant@openshiftdev origin]$ curl localhost:6061                                                           
Hello OpenShift!
```

Comment (liggitt) [inline examples/hello-openshift/README.md]: This is the pod running:
```
        "ports": [
          {
            "hostPort": 6061,
            "containerPort": 8080,
            "protocol": "TCP"
          }
        ],
```

Comment (mfojtik) [inline examples/hello-openshift/README.md]: hostPort...

**Comment by rhcarvalho** [comment]
[test]

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**